src/views/Dashboard/index.js

```diff
@@ -3,6 +3,7 @@ import {connect} from "react-redux";
 
 const mapStateToProps = (state) => ({
   userInfo: state.users.userInfo,
+  hasUser: Object.keys(state.users.userInfo).length > 0,
```

Comment:
can you create a `userSelector` for this?

good practice to have: **_If I have to write this more than once, write a function for it_**